tests/utils/test_raster_embedding.py

```diff
@@ -0,0 +1,61 @@
+# test_raster_embedding.py
```

Comment:
Good placeholders! Start writing some tests when you are ready: maybe wait until my next pull request though (I'm going to simplify the signature of raster_embedding() once this pull request goes through.)